connection_testcases.json

```diff
@@ -689,6 +689,15 @@
       "type": "multiple_compound_opts"
     }
   },
+  {
+    "opts": {
+      "credentials": "{\"port: 10000\"}",
```

Comment:
What does the `credentials` field represent here? I don't think that is a valid argument to `CreateClient()` is it?

Reply:
The `credentials` argument is treated the same way as `credentialsFile` minus reading the file.

Reply:
And `credentialsFile` is the credentials generated by the CLI when it creates an instance.  Not sure if we document it though.

Reply:
edgedb-go does not support `credentials`, but it does support `credentialsFile`. What is the use case for passing `credentials`? Why not just give each argument individually?

Reply:
We should not limit connection parameters to files.  If we expose `credentialsFile`, then plain `credentials` should also be supported.  That said, I'm not sure that we need to make `credentials`/`credentialsFile` public at all.

Reply:
To clarify, the use case is containers.  It is easy to pass values in environment variables.  Mounting files is not always easy or at all possible.

Reply:
Is this meant to be a string containing json? If so shouldn't it be `"{\"port\": 10000}"`